### PR TITLE
add netcdf output to Kinematic Model tests

### PR DESCRIPTION
**Pull request by trontrytel**
# Description

This adds netcdf output to Kinematic Model tests for microphysics. 

I have

- [x] Written and run all necessary tests with CLIMA by including `tests/runtests.jl`
- [x] Followed all necessary [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) and run `julia .dev/climaformat.jl .`
- [x] Updated the documentation to reflect changes from this PR.

# For review by CLIMA developers

- [ ] There are no open pull requests for this already
- [ ] CLIMA developers with relevant expertise have been assigned to review this submission
- [ ] The code conforms to the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) and has consistent naming conventions. `julia .dev/format.jl` has been run in a separate commit.
- [ ] This code does what it is technically intended to do (all numerics make sense physically and/or computationally)

### CODE REVIEW

**Review by charleskawczynski — APPROVED** [review]
Seems fine by me. @kpamnany everything look okay on the diagnostics side?

File: test/Atmos/Parameterizations/Microphysics/KM_ice.jl [hunk 1]
```diff
@@ -934,9 +935,37 @@ function main()
             nothing
         end
 
+    # output for netcdf
+    info = driver_config.config_info
```
Comment (kpamnany) [inline test/Atmos/Parameterizations/Microphysics/KM_ice.jl:939]: Is this being used?

**Comment by kpamnany** [comment]
Looks right to me, so if it works... :)

File: test/Atmos/Parameterizations/Microphysics/KM_ice.jl (hunk 1 above)
Comment (trontrytel) [reply test/Atmos/Parameterizations/Microphysics/KM_ice.jl]: It is not! Removed, thanks!

**Pull request merged**